tests/util/expect.ts

```diff
@@ -45,6 +46,24 @@ export async function expectOk<
   return block;
 }
 
+export function expectSuccessfulXCM(outcome: XcmV3TraitsOutcome) {
+  let error = "";
+  switch (true) {
+    case outcome.isError:
+      error = "Error executing XCM:" + outcome.asError.toString();
+      break;
+
+    case outcome.isIncomplete:
+      error = "Incomplete XCM:" + outcome.asIncomplete.toString();
+      break;
+
+    case outcome.isNone || outcome.isEmpty:
+      error = "Empty XCM";
+      break;
+  }
```

Comment:
might want to add a default case that throws error in case they add a new outcome in the future